draft-packages/split-button/KaizenDraft/SplitButton/styles.scss

```diff
@@ -265,51 +261,3 @@ $side-padding: 3/4 * $ca-grid;
   position: absolute;
   margin-top: 4px;
 }
-
```

Comment:
This deletion looks suss. Needs a double check.

Reply:
This was the wrong way around. The new draft should be copied to the old, but the old was copied to the new draft. I've checked that there weren't any recent changes in the old draft. Fix coming.